csrc/mmdeploy/apis/python/CMakeLists.txt

```diff
@@ -21,9 +21,13 @@ endforeach ()
 
 pybind11_add_module(${PROJECT_NAME} ${MMDEPLOY_PYTHON_SRCS})
 
-mmdeploy_load_static(${PROJECT_NAME} MMDeployStaticModules)
-mmdeploy_load_dynamic(${PROJECT_NAME} MMDeployDynamicModules)
-target_link_libraries(${PROJECT_NAME} PRIVATE MMDeployLibs)
+if (MMDEPLOY_BUILD_SDK_MONOLITHIC)
```

Comment:
I think there is no need to change this file

Reply:
When set MMDEPLOY_BUILD_SDK_MONOLITHIC to ON, the c api symbols that python use is no longer exists in MMDeployLibs but is in mmdeploy.